newIDE/app/src/UI/EmptyPlaceholder.js

```diff
@@ -43,7 +45,8 @@ export const EmptyPlaceholder = (props: Props) => (
             label={props.actionLabel}
```

Comment:
Consider if you could be using `LeftLoader` to display a loader next to a button (that's what we do, but maybe a loader inside the button is better)